edge-detect/coral/base_mqtt_pub_sub.py

```diff
@@ -0,0 +1,319 @@
+"""This file include the parent class for all future edge technology work by IQT Labs that
```

Comment:
@meadej - Is there a way to grab this file in the docker build process from the edgetech-core repo?  I'd like to not have a copy of this file outside of its source control

Reply:
Good point. Addressed in https://github.com/IQTLabs/edgetech-coral-detect/pull/1